client/src/main.rs

```diff
@@ -1657,14 +1657,30 @@ async fn main() {
                     );
 
                     ensure_payment(&api, &xt.encode().into(), tx_payment_cid_arg);
-                    let report = api.submit_and_watch_extrinsic_until(xt, XtStatus::Ready).unwrap();
-                    println!("Faucet created by {}. status: '{:?}'", who.public(), report.status);
+
+
+                    let result = api.submit_and_watch_extrinsic_until_success(xt, false);
+
+                    match result {
+                        Ok(report) => {
+                            let events = report
+                            .events
+                            .unwrap();
+
+                            println!("{}", AccountId::decode(&mut events[5].field_bytes()[0..32].as_ref()).unwrap().to_ss58check());
```

Comment:
thread 'main' panicked at 'range end index 32 out of range for slice of length 28', client/src/main.rs:1670:67
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

Reply:
faucet was created, though

Reply:
![image](https://github.com/encointer/encointer-node/assets/10366175/96d57289-e0d8-44e9-be36-bd2b9c521626)

Reply:
I think it is bad practice to use hardcoded indexing (especially for the event index)

Reply:
was an easy fix. committed directly

Reply:
thanks for fixing!